user/test/com/google/gwt/user/client/ui/FlexTableTest.java

```diff
@@ -150,5 +150,4 @@ public void secondarySetHeightTest() {
     cellFormatter.setHeight(3, 1, "300px");
     cellFormatter.setColSpan(3, 1, 2);
   }
```

Comment:
Add the newline again. Ideally HTMLTable and FlexTableTest should have no changes in this PR.